### PR TITLE
Updating tests after household and individual schema changes

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Selenium tests for census are failing after updating schemas. 
Removing page models

### How to review 
Selenium tests should pass

### CODE REVIEW

**Review by jonnyshaw89 — APPROVED** [review]

File: scripts/run_tests.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ if [ -z "$EQ_FUNCTIONAL_TEST_SUITES" ]; then
     if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
         export EQ_FUNCTIONAL_TEST_SUITES="core,census"
     else
-        export EQ_FUNCTIONAL_TEST_SUITES="core"
+        export EQ_FUNCTIONAL_TEST_SUITES="core,census"
```
Comment (jonnyshaw89) [inline scripts/run_tests.sh]: intentional?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: scripts/run_tests.sh (hunk 1 above)
Comment (benjefferies) [reply scripts/run_tests.sh]: Yeah for visibility that tests are fixed. I will remove when squashing before merge

File: tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js [hunk 2]
```diff
@@ -16,20 +15,19 @@ import NumberOfVehicles from '../../../pages/surveys/census/household/number-of-
 import HouseholdAndAccommodationCompleted from '../../../pages/surveys/census/household/household-and-accommodation-completed.page.js'
 import HouseholdMemberBegin from '../../../pages/surveys/census/household/household-member-begin.page.js'
 import DetailsCorrect from '../../../pages/surveys/census/household/details-correct.page.js'
-import Sex from '../../../pages/surveys/census/household/sex.page.js'
+import Over16 from '../../../pages/surveys/census/household/over-16.page.js'
 
 describe('Individual section Question 1', function () {
 
-  it('Given I am answering question 1 in the individual detail section, When I do not select any response, Then I am routed to What is you sex question 2 ', function () {
+  it('Given I am answering details correct question in the individual detail section, When I do not select any response, Then I am routed to What is you sex question 2 ', function () {
```
Comment (ajmaddaford) [inline tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js]: Is this description wrong now that it goes to the over 16 page?

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js (hunk 2 above)
Comment (benjefferies) [reply tests/functional/spec/census/issues/individual-Q1-are-these-details-correct.spec.js]: Good spot done

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**